spec/es_spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+require "logstash/devutils/rspec/spec_helper"
+require "ftw"
+require "logstash/plugin"
+require "logstash/json"
+require "stud/try"
+require "longshoreman"
+
+module ESHelper
+  def get_host
+    Longshoreman.new.get_host_ip
+  end
+
+  def get_port(protocol)
+    container = Longshoreman::Container.new
+    container.get("elasticsearch")
```

Comment:
I highly recommend using some kind of pre-defined constant that has something random in it instead of a statically named "elasticsearch."

The possibility of containers colliding because of identical naming is real.  This is why I assign a constant with a random bit in the Zabbix RSpec tests, e.g. `NAME = "logstash-output-zabbix-#{rand(999).to_s}"`, and then using `container.get(NAME)` and `@docker = Longshoreman.new("elasticsearch:1.6", NAME)`